sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/aio/client.py

```diff
@@ -40,16 +42,20 @@ def __init__(self, key: "Union[Key, str]", credential: "TokenCredential", **kwar
         if isinstance(key, Key):
             self._key = key
             self._key_id = parse_vault_id(key.id)
+            self._allowed_ops = frozenset(self._key.key_material.key_ops)
         elif isinstance(key, str):
             self._key = None
             self._key_id = parse_vault_id(key)
+            self._allowed_ops = None  # type: Optional[FrozenSet]
```

Comment:
Could an empty `frozenset` be sensibly interpreted the same as this case? (is there a way to spare downstream code from having to condition on whether the value is `None` or not)